### PR TITLE
Add DANIMESQ-XDAI space

### PR DESCRIPTION
**Pull request by DaniellMesquita**

### CODE REVIEW

File: spaces/daniell-mesquita-xdai/index.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "chainId": 100,
+  "decimals": 4,
+  "symbol": "DANIMESQ",
+  "defaultView": "all",
+  "address": "0x84d7fc44e4121adb74ef8d00a0f94cee7880d473",
+  "token": "0x84d7fc44e4121adb74ef8d00a0f94cee7880d473",
+  "core": ["0xDDfC2e10702d8A781727A34D83B3bb3CA94a3E91"],
+  "min": 80,
+  "invalid": []
+}
```
Comment (e-asia) [inline spaces/daniell-mesquita-xdai/index.json:13]: Something is wrong with the token, the call to query decimals() is failing. Is this a standard ERC20 token?

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/daniell-mesquita-xdai/index.json (hunk 1 above)
Comment (DaniellMesquita) [reply spaces/daniell-mesquita-xdai/index.json:13]: Probably I need to send it to XDAI to sync?

Comment (e-asia) [reply spaces/daniell-mesquita-xdai/index.json:13]: @bonustrack Could you have a look on that token?

Comment (bonustrack) [reply spaces/daniell-mesquita-xdai/index.json:13]: Looks good to me

**Review by e-asia — APPROVED** [review]

**Pull request merged**